lang/po/new/sk.po

```diff
@@ -0,0 +1,1767 @@
+#: ultralcd.cpp:4366
+msgid "Mode change in progress..."
+msgstr "Prebieha zmena modu..."
+
+# MSG_MODEL c=8
+#: messages.c:132
+msgid "Model"
+msgstr ""
+
+# MSG_NOZZLE_DIAMETER c=10
+#: messages.c:136
+msgid "Nozzle d."
+msgstr "Tryska"
+
+# MSG_GCODE_DIFF_CONTINUE c=20 r=4
+#: util.cpp:414
+msgid "G-code sliced for a different level. Continue?"
+msgstr "G-code pripraveny pre inu uroven. Pokracovat?"
+
+# MSG_GCODE_DIFF_CANCELLED c=20 r=7
+#: util.cpp:420
+msgid "G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+msgstr "G-code pripraveny pre inu uroven. Vygenerujte G-code znova. Tlac zrusena."
+
+# MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=5
+#: messages.c:134
+msgid "G-code sliced for a different printer type. Continue?"
+msgstr "G-code je pripraveny pre iny typ tlaciarne. Pokracovat?"
+
+# MSG_GCODE_DIFF_PRINTER_CANCELLED c=20 r=8
+#: messages.c:135
+msgid "G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+msgstr "G-code je pripraveny pre iny typ tlaciarne. Vygenerujte G-code znova. Tlac zrusena."
+
+# MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=5
+#: util.cpp:381
+msgid "G-code sliced for a newer firmware. Continue?"
+msgstr "G-code je pripraveny pre novsi firmware. Pokracovat?"
+
+# MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8
+#: util.cpp:387
+msgid "G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+msgstr "G-code je pripraveny pre novsi firmware. Prosim aktualizujte firmware. Tlac zrusena."
+
+# MSG_PREHEATING_TO_CUT c=20
+#: ultralcd.cpp:2309
+msgid "Preheating to cut"
+msgstr "Predhrev k strihu"
+
+# MSG_PREHEATING_TO_EJECT c=20
+#: ultralcd.cpp:2306
+msgid "Preheating to eject"
+msgstr "Predhrev k vysunutiu"
+
+# MSG_NOZZLE_DIFFERS_CONTINUE c=20 r=5
+#: util.cpp:294
+msgid "Printer nozzle diameter differs from the G-code. Continue?"
+msgstr "Priemer trysky tlaciarne sa lisi od G-code. Pokracovat?"
+
+# MSG_NOZZLE_DIFFERS_CANCELLED c=20 r=9
+#: util.cpp:301
+msgid "Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+msgstr "Priemer trysky tlaciarne sa lisi od G-code. Prosim skontrolujte nastavenie. Tlac zrusena."
+
+# MSG_SELFTEST_FS_LEVEL c=20
+#: ultralcd.cpp:8116
+msgid "%s level expected"
+msgstr "Ocakavany level %s"
+
+# MSG_RENAME c=18
+#: ultralcd.cpp:6500
+msgid "Rename"
+msgstr "Premenovat"
+
+# MSG_SELECT c=18
+#: ultralcd.cpp:6493
+msgid "Select"
+msgstr "Vybrat"
+
+# MSG_INFO_SENSORS c=18
+#: ultralcd.cpp:2041
+msgid "Sensor info"
+msgstr "Senzor info"
+
+# MSG_SHEET c=10
+#: messages.c:65
+msgid "Sheet"
+msgstr "Platna"
+
+# MSG_SOUND_BLIND c=7
+#: messages.c:146
+msgid "Assist"
+msgstr "Asist."
+
+# MSG_STEEL_SHEET c=18
+#: messages.c:66
+msgid "Steel sheets"
+msgstr "Platne"
+
+# MSG_Z_CORRECTION c=13
+#: ultralcd.cpp:5088
+msgid "Z-correct:"
+msgstr "Korekcia Z:"
+
+# MSG_Z_PROBE_NR c=14
+#: messages.c:149
+msgid "Z-probe nr."
+msgstr "Pocet merani Z"
+

```

Comment:
This is an incomplete po file missing the last message, please add ending with a new line.
```
# MSG_PRINTER_IP c=18
#: ultralcd.cpp:2028
msgid "Printer IP Addr:"
msgstr "IP adr. tlaciarne:"

```